tests/qa_automation/junit_log.pm

```diff
@@ -0,0 +1,142 @@
+# SUSE's openQA tests
```

Comment:
libraries should be in ../lib/directory

Reply:
little concern to put in lib directory, since this is only use for the qa_automation I think, but if you think still able be reused by other tests, then I am fine to move it to lib :)